doc/source/admin/scaling.md

```diff
@@ -553,10 +366,30 @@ as explained in [Job Handler Assignment Methods](#job-handler-assignment-methods
 </job_conf>
 ```
 
-As with statically defined handlers, `run.sh` will start the uWSGI process(es), but you will need to start the webless
-handler processes yourself. This is done on the command line like so (note the addition of the `--attach-to-pool`
+To let gravity know how many webless handler processes should be started set the `job_handler_count` value in galaxy.yml:
+
+```yaml
+gravity:
+  ...
+  job_handler_count: 4
+  ...
+```
+
+As with statically defined handlers, `run.sh` will start the process(es), but if you are not using `run.sh` or the generated supervisor config you will need to start the webless handler processes yourself. This is done on the command line like so (note the addition of the `--attach-to-pool`
 option):
 
+TODO: `job_handler_count` doesn't really make sense if you have multiple pools. Could be
+
+```yaml
+gravity:
+  ...
+  job_handler_count:
+    _default_: 4
+    special: 2
+```
+
+?
```

Comment:
If you have other idea let me know!

Reply:
Maybe if it's an integer it's assumed to be `_default_` and if it's a dict it works as you've shown it here? Also doesn't handle workflow schedulers currently.